contrib/python-bindings/notebooks/index.ipynb

```diff
@@ -8,7 +8,11 @@
     "This page contains a link to every notebook as well as a short explanation on what the notebooks do.\n",
     "\n",
     "## tutorial-1\n",
-    "[tutorial-1](https://github.com/dealii/dealii/tree/master/contrib/python-bindings/notebooks/tutorial-1.ipynb) shows how to create a **Triangulation** using the python binding. We also show how to refine the grid, merge two **Triangulations**, and finally how to output the **Triangulation** so that it can be loaded in a C++ code."
+    "[tutorial-1](https://github.com/dealii/dealii/tree/master/contrib/python-bindings/notebooks/tutorial-1.ipynb) shows how to create a **Triangulation** using the python binding. We also show how to refine the grid, merge two **Triangulations**, and finally how to output the **Triangulation** so that it can be loaded in a C++ code.\n",
+    "\n",
+    "## step-62\n",
+    "[step-62](https://github.com/dangars/dealii/tree/phononic-cavity/examples/step-62/step-62.ipynb) shows how to to calculate the [energy band gap](https://en.wikipedia.org/wiki/Band_gap) and the\n",
+    "mechanical resonance of a [micropillar superlattice cavity](https://doi.org/10.1103/PhysRevA.94.033813).\n"
```

Comment:
I had a [conversation](https://github.com/dealii/dealii/pull/6747#issuecomment-411889703) with @Rombur about how to organize the notebooks. We agreed to to add an entry for each notebook in `index.ipynb`. In addition notebooks of the tutorials will be referenced by a symbolic link, see below. You can take a look to the conversation [conversation](https://github.com/dealii/dealii/pull/6747#issuecomment-411889703) and let us know if that is ok for you.

Reply:
Ah, ok. I don't know anything about the ipnb files, and only thought that this was a mistake because none of the other programs are listed there. This works for me if you think it's right this way!

Reply:
Before we merge this PR, we need to change the link to `https://github.com/dangars` to `https://github.com/dealii`

Reply:
@Rombur Thanks for seeing this. I updated the link from https://github.com/dangars to https://github.com/dealii I did a grep to make sure that there a no other links that point to dangars